pkg/operator/ceph/object/rgw.go

```diff
@@ -365,3 +365,8 @@ func emptyPool(pool cephv1.PoolSpec) bool {
 func BuildDomainName(name, namespace string) string {
 	return fmt.Sprintf("%s-%s.%s", AppName, name, namespace)
 }
+
+// buildDNSEndpoint build the dns name to reach out the service endpoint
+func buildDNSEndpoint(domainName string, port int32) string {
+	return fmt.Sprintf("http://%s:%d", domainName, port)
```

Comment:
what about if the rgw server only support `https`? Or it is only consumed for independent mode and ssl is not recommended for independent mode?